test/karma/karma.conf.js

```diff
@@ -86,7 +87,8 @@ module.exports = function( config ) {
 		captureTimeout: 3e5,
 		browserNoActivityTimeout: 3e5,
 		browserDisconnectTimeout: 3e5,
-		browserDisconnectTolerance: 3
+		browserDisconnectTolerance: 3,
+		startTunnel: false
```

Comment:
Please add an empty line above this one and a line with a comment saying the tunnel is started manually in `.travis.yml` for stability reasons.